src/H5.c

```diff
@@ -214,14 +215,14 @@ H5_init_library(void)
      */
     if (!H5_dont_atexit_g) {
 
-#if defined(H5_HAVE_THREADSAFE)
+#ifdef H5_HAVE_THREADSAFE_API
```

Comment:
Example of macro name change.  H5_HAVE_THREADSAFE_API if either H5_HAVE_THREADSAFE or H5_HAVE_CONCURRENCY is enabled